api/v1/utils/roles.js

```diff
@@ -0,0 +1,10 @@
+var _ = require('lodash');
+
+var ALL_ROLES = ['ADMIN', 'STAFF', 'SPONSOR', 'MENTOR', 'VOLUNTEER', 'HACKER'];
```

Comment:
@arnmishra: Make sure that this is comprehensive (maybe also ask Koroush and Niki), (do you want to distingush between Open source and Sponsor Mentors?)